server/controllers/cohort.js

```diff
@@ -147,24 +149,24 @@ const getCohortDetails = (req, resp) => {
       }
 
       const {
-        ownerIds,
+        ownerIds: ownersCollection,
         _id,
         isArchived,
         description,
         name,
-        memberIds: membersData
+        memberIds: membersCollection
       } = doc;
 
       if (isArchived) {
         return resp.status(200).json({ _id, isArchived, name });
       }
 
-      const owners = ownerIds.map(owner => owner.id);
+      const ownerIds = ownersCollection.map(owner => owner.id);
+      const isOwner = checkIfArrayContainsUserId(ownerIds, req.user._id);
       const members = responseWithCohortMembers(
-        [...membersData, ...ownerIds],
-        owners
+        [...membersCollection, ...ownersCollection],
```

Comment:
this could be only members (owners are already there)

Reply:
I need the whole collections of data here to respond with members with data like display_name etc...

Reply:
but memberIds contains all ownerIds, if you populate memberIds there is no need to populate the same data for ownerIds. OwnerIds is needed to check roles only